source/transactions/tests/mongos-recovery-token.yml

```diff
@@ -0,0 +1,362 @@
+database_name: &database_name "transaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+
+tests:
+  - description: commitTransaction explicit retries succeed on new mongos
```

Comment:
The test doesn't assert what the description says.  What are we really trying to test here?  That the recovery token is included on all commit attempts?  That it's the same for every attempt?

Reply:
Yeah it's just supposed to test that explicitly retrying commit succeeds. Renamed to simply "commitTransaction explicit retries succeed".